react/react1/week3/todo-app/src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+import { fetchData } from "./Utils";
+import Container from "./components/Container/Container";
+import FormTodoList from "./components/FormTodoList/FormTodoList";
+import Header from "./components/Header/Header";
+import TodoList from "./components/TodoList/TodoList";
+
+function App() {
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => {
+    const fetch = async () => {
```

Comment:
Good. useEffects can't be async, so we have to call an async function from inside our useEffect